### ISSUE TITLE
Copy & Paste bug in VS Code

### ISSUE THREAD

**Issue by clovett** [Microsoft/vscode#53918]
Issue Type: <b>Bug</b>

If I click on the mardown preview window, double click a word and extend the selection to include a bit more, then hit "Copy" it copies the entire line onto the clipboard rather than just the text I selected.  If I select the text without using double click then it works fine.

VS Code version: Code 1.24.1 (24f62626b222e9a8313213fb64b10d741a326288, 2018-06-13T17:51:32.889Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz (8 x 3510)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.95GB (21.80GB free)|
|Process Argv|d:\Program Files\Microsoft VS Code\Code.exe .|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
githistory|don|0.4.1
python|ms-|2018.6.0
cpptools|ms-|0.17.6
vscode-arduino|vsc|0.2.17

</details>

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code build](https://code.visualstudio.com/insiders/) with all extensions disabled?

**Comment by lovettchris** [comment]
Yes, I upgraded to 1.25, and it happens on this mardown paragraph:

## Create your training and validation datasets
Before you retarget an existing neural network, you must create a training dataset and a validation dataset for the image classes you want to recognize. In this tutorial, you'll classify images of 10 common fruits, including `apple, banana, blackberry, blueberry, cherry, grapes, lemon, lime, orange and raspberry`.

View this in the markdown preview, and double click blueberry.  You will see "blueberry" selected, but now copy and paste and you will see the entire paragraph get pasted, not just the selected word "blueberry".
